src/app/constants/urls.js

```diff
@@ -0,0 +1,2 @@
+export const BASE_URL = 'https://wbooks-api-stage.herokuapp.com/api/v1';
```

Comment:
Usually we dont want to have this kind of information into our project. So we put the BASE_URL and any sensitive key into a `.env` file on the root of project.
add this and then add .env file into `.gitignore` file to avoid upload to the repo